Update on README, UserGuide (#5)
openshift/kube-compare: README.md

### Workflow Description

To Compare a known valid reference configuration with a live cluster:

`kubectl cluster-compare -r <referenceConfigurationDirecotry>`

To Compare a known valid reference configuration with a local set of CRs:

`kubectl cluster-compare -r <referenceConfigurationDirecotry> -f <inputConfiguration>`

> To Compare a known valid reference configuration with a live cluster and with a user config:

Reviewer: Because the tool is comparing a reference with the configuration on a cluster, I think we need to clarify the term "user config" here to make it obvious that this is a file which is tuning the behavior of the tool and not part of what is being compared.

`kubectl cluster-compare -r <referenceConfigurationDirecotry> -c <userConfig>`

> To Run a known valid reference configuration with a must-gather output:
> 
> `kubectl cluster-compare -r <referenceConfigurationDirecotry> -f "must-gather*/*/cluster-scoped-resources","must-gather*/*/namespaces" -R`

Reviewer: Trying to keep this generic:
```suggestion
To Run a known valid reference configuration against a support archive:

`kubectl cluster-compare -r <referenceConfigurationDirecotry> -f "must-gather*/*/cluster-scoped-resources","must-gather*/*/namespaces" -R`
```
